### PR TITLE
Configurable unfurler config

### PR DESCRIPTION
**Pull request by knorrium**
Allow the unfurler to read from different config paths in preparation for dockerization

### CODE REVIEW

File: unfurler/src/config.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
-const configFile = require('../config.json');
+const fs = require('fs');
+const path = require('path');
+
+const configPath = process.env.UNFURLER_CONFIG || '../config.json';
```
Comment (mononaut) [inline unfurler/src/config.ts:4]: can we make the default path `'./config.json'`?

(`require('../config.json')` is relative to this file, but `path.resolve('../config.json')` is relative to the working directory, i.e. one level higher)

**Review by mononaut — CHANGES_REQUESTED** [review]
Tested ACK, pending change to the default config path

File: unfurler/src/config.ts (hunk 1 above)
Comment (knorrium) [reply unfurler/src/config.ts:4]: @mononaut good catch. I think I changed this when it was running off `dist`

**Review by mononaut — APPROVED** [review]
Tested ACK @ e9e507af

**Pull request merged**